open_spiel/python/algorithms/projected_replicator_dynamics.py

```diff
@@ -73,6 +73,48 @@ def _project_distribution(updated_strategy, gamma):
   return updated_strategy
 
 
+
+def _project_distribution(updated_strategy, gamma):
```

Comment:
I suggest renaming this function to `_approx_simplex_projection` and your contribution to `_simplex_projection`. Could you also add an argument (e.g. `use_approx=False`) to `projected_replicator_dynamics` that allows to choose the approximate version but defaults to the actual L2 projection.